bin/lib/ce_install.py

```diff
@@ -148,6 +148,7 @@ def squash_mount_check(rootfolder, subdir, context):
 )
 @click.option("--enable", metavar="TYPE", multiple=True, help='Enable targets of type TYPE (e.g. "nightly")')
 @click.option("--only-nightly", is_flag=True, help="Only install the nightly targets")
+@click.option("--only-native-aarch64", is_flag=True, help="Only install the Native Aarch64 targets")
```

Comment:
Sounds like we probably need a more general `--only` if we need to start adding lots of these.